wgs.yaml

```diff
@@ -96,6 +96,71 @@ workinggroups:
   - name: katib
     owners:
     - https://raw.githubusercontent.com/kubeflow/katib/master/OWNERS
+- dir: wg-notebooks
+  name: Notebooks
+  mission_statement: >
+    WG Notebooks is responsible for the user experience around Notebooks and
+    their integrations with Kubeflow.
+
+  charter_link: charter.md
+  label: area/wg-notebooks
+  leadership:
+    chairs:
+    - github: elikatsis
```

Comment:
Where did you land on adding @thesuperzapper as a lead?

Reply:
I'm adding him as a lead and tech lead.
Waiting for the `make generate` issue to be resolved #415

Reply:
@kimwnasptd you're waiting for #415 to be resolved before adding him to wgs.yaml?

I don't think we can merge this until #415 is fixed so you might want to volunteer to fix it.